python3/packages/observer.py

```diff
@@ -288,12 +289,13 @@ def instrument_function(wrapped, _, args, kwargs):
                     # class or classmethod
                     aspan.set_attribute("xs.span.args.str", str(args))
                     aspan.set_attribute("xs.span.kwargs.str", str(kwargs))
-                else:
-                    # function, staticmethod or instancemethod
+                elif type(wrapped)==types.FunctionType or type(wrapped)==types.MethodType:
```

Comment:
A tuple, as in `isinstance(x, (A, B))`, may be given as the target to check against.
This is equivalent to `isinstance(x, A) or isinstance(x, B)`:

```suggestion
                elif type(wrapped)==types.FunctionType or type(wrapped)==types.MethodType:
                elif isinstance(wrapped, (types.FunctionType, types.MethodType)):
```